api/envoy/service/extension/v3/config_discovery.proto

```diff
@@ -18,11 +18,12 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: Extension config discovery service (ECDS)]
 
 // A service that supports dynamic configuration updates for a specific filter.
-// Currently, ECDS is supported for HTTP filters and Listener filters. Please check
-// :ref:`Extension Config Discovery Service (ECDS) API <config_overview_extension_discovery>`.
+// Currently, ECDS is supported for downstream network filters, HTTP filters and Listener filters.
```

Comment:
High level comment:
Can you point me to Envoy's support for upstream network filters?
If it is not supported, I suggest removing the unused-code/comments as they may cause more confusion.

Reply:
I see. I added this in advance in order to split PRs logic instead of having PRs with big source change. I'll add upstream network filter support soon so it won't be stale much longer.